docs/pages/push-notifications/fcm-credentials.mdx

```diff
@@ -0,0 +1,167 @@
+---
+title: Add Google Service Account Keys using FCM V1
+sidebar_title: Add Android FCM V1 credentials
+description: Learn how to create or use a Google Service Account Key for sending Android Notifications using FCM V1.
+---
+
+import { Step } from '~/ui/components/Step';
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+import { Tab, Tabs } from '~/ui/components/Tabs';
+
+## Create a new Google Service Account Key
+
+Here are the steps to configure a new Google Service Account Key in EAS for sending Android Notifications using FCM V1.
+
+<Step label="1">
+  Create a new Firebase project for your app in the [Firebase Console](https://console.firebase.google.com). If you already have a Firebase project for your app, continue to the next step.
+
+  <ImageSpotlight
+    alt="Create a project in the Firebase Console"
+    src="/static/images/creating-google-service-account/fcm-v1/new-service-account/01-new-firebase-project.png"
+    style={{ maxWidth: 800 }}
+  />
+</Step>
+
+<Step label="2">
+  In the Firebase console, open **Project settings** > [**Service accounts**](https://console.firebase.google.com/project/_/settings/serviceaccounts/adminsdk) for your project.
+
+  <ImageSpotlight
+    alt="Service accounts settings tab in Firebase console"
+    src="/static/images/creating-google-service-account/fcm-v1/new-service-account/02-manage-service-accounts.png"
+    style={{ maxWidth: 800 }}
+  />
+</Step>
+
+<Step label="3">
+  Click **Generate New Private Key**, then confirm by clicking **Generate Key**. Securely store the JSON file containing the private key.
+
+  <ImageSpotlight
+    alt="Generate a new private key in the Firebase Console"
+    src="/static/images/creating-google-service-account/fcm-v1/new-service-account/03-generate-key.png"
+    style={{ maxWidth: 800 }}
+  />
+</Step>
+
+<Step label="4">
+  Upload the JSON file to EAS and configure it for sending Android notifications. This can be done using EAS CLI or in [Expo dashboard](https://expo.dev).
+
+  <Tabs tabs={["EAS CLI", "expo.dev"]}>
+    <Tab>
+      - Run `eas credentials`
+      - select `Android`
+      - select `production`
+      - select `Google Service Accounts`
+      - and select `FCM V1`
+    </Tab>
+    <Tab>
+      - Under **Project settings**, click [Credentials](https://expo.dev/accounts/[account]/projects/[project]/credentials) in the navigation menu
+      - For **Android**, click **Add Application Identifier** or select an existing **Application identifier**
+      - Under **Service Credentials**, find **FCM V1 service account key**
+      <ImageSpotlight
+        alt="Project credentials page on expo.dev"
+        src="/static/images/creating-google-service-account/fcm-v1/new-service-account/04-upload-credential-1.png"
+        style={{ maxWidth: 800 }}
+      />
```

Comment:
```suggestion

      <ImageSpotlight
        alt="Project credentials page on expo.dev"
        src="/static/images/creating-google-service-account/fcm-v1/new-service-account/04-upload-credential-1.png"
        style={{ maxWidth: 800 }}
      />
```

This will fix the CI failure. It was caused by my suggestion here earlier.